pepper-apis/src/test/resources/sendgrid-event-testdata/sendgrid-event-testdata.json

```diff
@@ -0,0 +1,138 @@
+[
+  {
+      "email":"example1@test.com",
```

Comment:
I recommend using `@datadonationplatform.org` for dummy emails.

Reply:
Fixed